openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```
@@ -445,165 +323,208 @@ public static Builder builder() {
+                            + "after catching an unrelated error", ex);
+                }
+                throw th;
+            }
         }
 
-        /**
-         * Sets whether annotation processing for OpenAPI annotations should
-         * be enabled.
-         *
-         * @param isEnabled whether annotation processing should be turned on
-         * @return updated builder instance
-         */
-        public Builder enableAnnotationProcessing(boolean isEnabled) {
-            isAnnotationProcessingEnabled = isEnabled;
-            return this;
+        private OpenApiStaticFile getDefaultStaticFile() {
+            final List<String> candidatePaths = LOGGER.isLoggable(Level.FINER) ? new ArrayList<>() : null;
+            for (OpenAPIMediaTypes candidate : OpenAPIMediaTypes.values()) {
+                for (String type : candidate.matchingTypes()) {
+                    String candidatePath = DEFAULT_STATIC_FILE_PATH_PREFIX + type;
+                    InputStream is = null;
+                    try {
+                        is = getContextClassLoader().getResourceAsStream(candidatePath);
+                        if (is != null) {
+                            Path path = Paths.get(candidatePath);
+                            LOGGER.log(Level.FINE, () -> String.format(
+                                    OPENAPI_DEFAULTED_STATIC_FILE_LOG_MESSAGE_FORMAT,
+                                    path.toAbsolutePath().toString()));
+                            return new OpenApiStaticFile(is, candidate.format());
+                        }
+                        if (candidatePaths != null) {
+                            candidatePaths.add(candidatePath);
+                        }
+                    } catch (Throwable th) {
+                        if (is != null) {
+                            try {
+                                is.close();
+                            } catch (IOException ex) {
+                                LOGGER.log(Level.FINE,
```

Consider adding a suppressed throwable instead of logging.

---

Done.